### PR TITLE
Ldap documentation

### PR DESCRIPTION
**Pull request by dcrissman**
Adds documentation for LDAP.

### CODE REVIEW

**Comment by jewzaam** [comment]
You're missing an update to SUMMARY.md
Are you creating this by hand?  If so, check out https://github.com/GitbookIO/editor

**Comment by dcrissman** [comment]
@jewzaam Yes, by hand with the help of an Eclipse plugin. I downloaded Gitbook for next time. In the meantime, change has been pushed up and is ready for review.

File: language_specification/ldap.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+As with other plugins, the metadata is made up of two parts, the _entityInfo_ and _schema_. See the documentation on Metadata for more high level details on how these are used. Continue reading for LDAP specifics.
+
+More examples can be found in the **lightblue-ldap-integration-test** module.
+
+### entityInfo
+
+**Attributes:**
+* **backend** - Name of the datasource provider to use. Should always be "ldap".
+* **database** - Name of database to associate this entity too. Should match the _database_ attribute in _datasources.json_.
+* **basedn** - The base _dn_ for where the entities should live within the LDAP directory structure.
```
Comment (jewzaam) [inline language_specification/ldap.md]: `basedn` has the "d" lower case.
`bindableDn` has the "d" upper case.
Let's be consistent.

**Comment by jewzaam** [comment]
Unless the word "lightblue" begins a sentence it should be lower case.

I recommend using sub-sections to break this into a few different documents:
* Configure
* Metadata
* CRUD Operation Examples

General Notes could be kept in the higher level "LDAP" section.

**Comment by dcrissman** [comment]
@jewzaam ready for review

**Comment by jewzaam** [comment]
@bserdar @derek63 @luan-cestari tagging to say I like how this documentation is broken down, makes it easy to consume and answers these questions:
* how do I set it up?
* how do I create metadata?
* how do I query for data?

**Pull request merged**